### ISSUE TITLE
Add new shift time section into configuration YAML

### ISSUE THREAD

**Issue by HeavyWombat** [homeport/pd#2]
For components that require 24h coverage for 7 days (24/7), it is not unusual to split the work between different teams in different geographical locations. One option could be with one team in East Asia, one in Europe, and another one in East Coast America. This way you can split the work and each team has somehow acceptable working times.

Since each project has different locations, different teams, and different preferences, it is required that the settings for the shift times can be configured.

Example:
```yaml
---
authtoken: foobar
shift-times:
- name: Team Asia
  start: 23:00
  end: 07:00
- name: Team Europe
  start: 07:00
  end: 15:00
- name: Team America
  start: 15:00
  end: 23:00
```

This should configure the `pd` tool to know that the project uses three teams to cover 24h per day. Please note: It is absolutely okay to be configured in a way that some times are not covered. All times are meant to be in Zulu time/UTC so that there is no confusion regarding what timezone is meant.

The goal is that `pd` is reading the new YAML input. A new command called `current-shift` should be added via `cobra add ...` and it should look up the current user current time to print out which shift is currently active.

_Acceptance Criteria:_ As a user, I want to call `pd current-shift` to get the following output:
```sh
$ pd current-shift
At the moment, Team Europe is in charge.
```

**Comment by HeavyWombat** [comment]
For components that require 24h coverage for 7 days (24/7), it is not unusual to split the work between different teams in different geographical locations. One option could be with one team in East Asia, one in Europe, and another one in East Coast America. This way you can split the work and each team has somehow acceptable working times.

Since each project has different locations, different teams, and different preferences, it is required that the settings for the shift times can be configured.

Example:
```yaml
---
authtoken: foobar
shift-times:
- name: Team Asia
  start: 23:00
  end: 07:00
- name: Team Europe
  start: 07:00
  end: 15:00
- name Team America
  start: 15:00
  end: 23:00
```

This should configure the `pd` tool to know that the project uses three teams to cover 24h per day. Please note: It is absolutely okay to be configured in a way that some times are not covered. All times are meant to be in Zulu time/UTC so that there is no confusion regarding what timezone is meant.

The goal is that `pd` is reading the new YAML input. A new command called `current-shift` should be added via `cobra add ...` and it should look up the current user current time to print out which shift is currently active.

_Acceptance Criteria:_ As a user, I want to call `pd current-shift` to get the following output:
```sh
$ pd current-shift
At the moment, Team Europe is in charge.
```

**Comment by HeavyWombat** [comment]
Fixed by #10.
